core/src/data/value/mod.rs

```diff
@@ -555,6 +555,53 @@ impl Value {
     pub fn to_cmp_be_bytes(&self) -> Result<Vec<u8>> {
         self.try_into().map(|key: Key| key.to_cmp_be_bytes())
     }
+
+    /// Support only [`Value::Str`] variant
+    ///
+    /// Returns the position where the first letter of the substring starts if the string contains a substring.
+    ///
+    /// Returns `I64(0)` if the string to be found is not found.
+    /// Returns minimum value `I64(1)` when the string is found.
+    /// Returns Null if NULL parameter found.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use gluesql_core::{
+    ///     data::ValueError,
+    ///     prelude::Value::*
+    /// };
```

Comment:
It is very rare for users to use `gluesql_core`, and it would be good to roll back because they will normally use `gluesql`.